scripts/pre_push_hook.py

```diff
@@ -200,6 +221,13 @@ def _start_sh_script(scriptname):
     return task.returncode
 
 
+def _is_in_dirty_state():
+    """Returns true if the repos state is dirty (modified files are uncommited)
+    ignores untracked files."""
```

Comment:
s/repos/repo's
Add full stop after first line, and s/ignores/Ignores (new sentence)